### DISCUSSION TITLE
What is the meaning of "gas" as opposed to "transaction cost" and "execution cost" when Remix reports overall gas expense?

### DISCUSSION THREAD

**Discussion by reselbob** [remix-run/remix#4874] [Q&A] [answered]
Hi: When I Deploy or execute a function in Remix I get three categories of cost:

- gas
- transaction cost
- execution cost

For example:
```
gas:    32296 gas
transaction cost:   28083 gas 
execution cost: 28083 gas 
```
I understand what transaction and execution costs are, but don't know what the first gas value means or how it is calculated.

Help?

I am attaching a screenshot that illustrates what I mean.

Thanks in advance!
![fees-question](https://user-images.githubusercontent.com/1110569/207778866-df6f9e17-7274-4da7-9781-1888059e990f.png)

**Comment by machour** [comment, marked as answer]
You're mistaking this repository for the Remix IDE repository.
